src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
 }
+
+html {
+  --blue-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  text-align: center;
+  display: block;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  padding: 23px 0;
```

Comment:
<img width="681" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/7ab39145-bf99-4138-a8e8-3bf4d5488752">
try to rewrite it with `height` and center using the flex